app/ante/handler_options.go

```diff
@@ -79,14 +81,12 @@ func newCosmosAnteHandler(options HandlerOptions) sdk.AnteHandler {
 	return sdk.ChainAnteDecorators(
 		ethante.RejectMessagesDecorator{}, // reject MsgEthereumTxs
 		ante.NewSetUpContextDecorator(),
-		ante.NewRejectExtensionOptionsDecorator(),
```

Comment:
This decorator should be replaced with `NewExtensionOptionsDecorator`
https://github.com/cosmos/cosmos-sdk/blob/v0.46.1/x/auth/ante/ext.go#L35

Reply:
just added there.